integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanDisbursementDetailsIntegrationTest.java

```diff
@@ -347,6 +350,83 @@ public void createApproveAndValidateMultiDisburseLoan() throws ParseException {
         this.editLoanDisbursementDetails();
     }
 
+    @Test
```

Comment:
Missing test case:
1. Create and disburse loan product with "disallowExpectedDisbursements=true"
2. Undo the disbursement
3. Disburse again

Reply:
Done